crmsh/bootstrap.py

```diff
@@ -205,6 +205,9 @@ class SBDManager(object):
     DISKLESS_SBD_WARNING = """Diskless SBD requires cluster with three or more nodes.
 If you want to use diskless SBD for two-nodes cluster, should be combined with QDevice."""
     PARSE_RE = "[; ]"
+    DISKLESS_CRM_CMD = "crm configure property stonith-enabled=true stonith-watchdog-timeout={} stonith-timeout={}"
+    SBD_WATCHDOG_TIMEOUT_DEFAULT = 35
```

Comment:
It'd be more clear to use a name like `SBD_WATCHDOG_TIMEOUT_DEFAULT_WITH_QDEVICE` :-)

Reply:
Changed